python/cli/prelude_cli/views/build.py

```diff
@@ -42,22 +42,41 @@ def create_test(controller, name, unit, test, techniques, advisory):
         )
 
     if not test:
+        """ Mark-up, upload, and write the test file template. """
         basename = f'{t["id"]}.go'
-        template = pkg_resources.read_text(templates, 'template.go')
-        template = template.replace('$ID', t['id'])
-        template = template.replace('$NAME', name)
-        template = template.replace('$UNIT', unit or '')
-        template = template.replace('$CREATED', str(datetime.utcnow()))
+        utc_time = str(datetime.now(timezone.utc))
+        test_template = pkg_resources.read_text(templates, 'template.go')
+        test_template = test_template.replace('$ID', t['id'])
+        test_template = test_template.replace('$NAME', name)
+        test_template = test_template.replace('$UNIT', unit or '')
+        test_template = test_template.replace('$CREATED', utc_time)
         
         with Spinner(description='Applying default template to new test'):
-            controller.upload(test_id=t['id'], filename=basename, data=template.encode('utf-8'))
+            controller.upload(test_id=t['id'], filename=basename, data=test_template.encode('utf-8'))
             t['attachments'] = [basename]
 
         test_dir = PurePath(t['id'], basename)
         Path(t['id']).mkdir(parents=True, exist_ok=True)
         
         with open(test_dir, 'w', encoding='utf8') as test_code:
-            test_code.write(template)
+            test_code.write(test_template)
+
+        print_json(data=t)
+
+        """ Mark up, upload, and write the test README template. """
+        readme_template = pkg_resources.read_text(templates, 'README.md')
```

Comment:
This block is so similar to the block above, it'd be nice to have an inner func to dedupe the code so the "main" code can just be

```
create_attachement(template='template.go', name=basename)
create_attachement(template='README.md', name='README.md')
```

Reply:
eh, i'm going to punt on this for now. i agree it may be nice to have a helper func to tidy this section up, but don't think it's strictly necessary at this very moment.

Reply:
I very much take the other view ha unless there is a _very_ good reason to not have clean code, default to doing it the right way the first time

Our code base right now is super slim, has next to no tech debt, and I'd love to keep it that way as long as possible